lib/cli/src/frameworks/vue/Page.vue

```diff
@@ -2,9 +2,9 @@
   <article>
     <my-header
       :user="user"
-      @onLogin="onLogin"
-      @onLogout="onLogout"
-      @onCreateAccount="onCreateAccount"
+      @login="user = { name: 'Jane Doe' }"
```

Comment:
these methods do not exist in `my-header`. Can you please update them to  `onLogin`, `onLogout` and `onCreateAccount`? And maybe move the inline code changes to methods?